.github/workflows/dataset-download-metrics.yml

```diff
@@ -0,0 +1,37 @@
+name: "Dataset Download Metrics"
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+
+jobs:
+  aggregate-metrics:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: |
+          pip install -r requirements.txt
+      - name: Configure Prod AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_PROD_ROLE_TO_ASSUME }}
+          role-duration-seconds: 900
+      - name: Run metrics aggregation script
+        run: |
+          python3 scripts/athena.py | tee dataset_metrics.txt
+      - name: Write to Slack on success
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        run: |
+          DOWNLOAD_STATS=$(cat dataset_metrics.txt)
+          METRICS_MSG="\`\`\`PROD DATASET DOWNLOAD METRICS: \n${DOWNLOAD_STATS}\`\`\`"
```

Comment:
I think slack limits the size of posted messages and this ends up getting split up. To maintain the formatting you may want to split it into ~30 datasets per post request

Reply:
I actually did a few other things to shorten this:
- Consolidated the ID to be the collection id + dataset id.. it was separating out by file type before.
- Removed private collections and private datasets from getting counted
- Set a one week time frame instead of all time.

These changes should make it such that it is less than 30 datasets for now. I'll revisit if it becomes more later. Is this OK with you?